lib/mix/tasks/phx.gen.context.ex

```diff
@@ -93,7 +93,9 @@ defmodule Mix.Tasks.Phx.Gen.Context do
     live: :boolean,
     compile: :boolean,
     primary_key: :string,
-    migration: :boolean
+    migration: :boolean,
+    scope: :string,
+    no_scope: :boolean
```

Comment:
I believe you don't need `no_scope`. Since you have `--scope`, you can pass `--no-scope`, and it will come with `scope: nil`. You can distinguish between them verifying if the option exists or not.

Reply:
Hmm
```
iex(1)> OptionParser.parse(["--no-scope"], switches: [scope: :string])
{[], [], []}
iex(2)> OptionParser.parse(["--scope"], switches: [scope: :string])
{[], [], [{"--scope", nil}]}
iex(3)> OptionParser.parse(["--scope", "foo"], switches: [scope: :string])
{[scope: "foo"], [], []}
```

Reply:
Sad trombone. You are right. You could explicitly check for `"--no-scope" in args` and act accordingly, but then I guess it would end-up the same.